plugins/events/create.js

```diff
@@ -159,9 +162,11 @@ module.exports = () => ({
                                         payload.prInfo = prInfo;
                                         payload.prRef = prInfo.ref;
                                         payload.chainPR = pipeline.chainPR;
+                                        const restrictPR = pipeline.annotations[ANNOT_RESTRICT_PR];
 
-                                        // PR author should be able to rerun their own PR build
-                                        if (prInfo.username === username) {
+                                        // PR author should be able to rerun their own PR build if restrictPR is not on
```

Comment:
there is a value `fork` also for restrictPR

Reply:
can you elaborate on this?

Reply:
https://docs.screwdriver.cd/user-guide/configuration/annotations#pipeline-level-annotations